c7n/resources/dynamodb.py

```diff
@@ -450,3 +454,235 @@ class resource_type(object):
+                        'InvalidParameterCombinationException',
+                        'InvalidParameterValueException'):
+                    self.log.warning(
+                        'Exception tagging %s: \n%s' % (
+                            r['ClusterName'], e))
+                    continue
+                raise
+
+
+@DynamoDbAccelerator.action_registry.register('remove-tag')
+class DaxRemoveTagging(RemoveTag):
+    """Action to remove tag(s) on a resource
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: dax-remove-tag
+            resource: dax
+            filters:
+              - "tag:OutdatedTag": present
+            actions:
+              - type: remove-tag
+                tags: ["OutdatedTag"]
+    """
+    permissions = ('dax:UntagResource',)
+
+    def process_resource_set(self, resources, tag_keys):
+        client = local_session(
+            self.manager.session_factory).client('dax')
+        for r in resources:
+            try:
+                client.untag_resource(
+                    ResourceName=r['ClusterArn'], TagKeys=tag_keys)
+            except ClientError as e:
+                if e.response['Error']['Code'] in (
+                        'ClusterNotFoundFault',
+                        'InvalidARNFault',
+                        'InvalidClusterStateFault',
+                        'InvalidParameterCombinationException',
+                        'InvalidParameterValueException',
+                        'TagNotFoundFault'):
+                    self.log.warning(
+                        'Exception removing tags on %s: \n%s' % (
+                            r['ClusterName'], e))
+                    continue
+                raise
+
+
+@DynamoDbAccelerator.action_registry.register('mark-for-op')
+class DaxMarkForOp(TagDelayedAction):
+    """Action to specify an action to occur at a later date
+
+    :example:
+
+    .. code-block:: yaml
+
+        policies:
+          - name: dax-mark-tag-compliance
+            resource: dax
+            filters:
+              - "tag:custodian_cleanup": absent
+              - "tag:OwnerName": absent
+            actions:
+              - type: mark-for-op
+                tag: custodian_cleanup
+                msg: "Missing tag 'OwnerName': {op}@{action_date}"
+                op: delete
+                days: 7
+    """
+    permission = ('dax:TagResource',)
+
+    def process_resource_set(self, resources, tags):
+        client = local_session(self.manager.session_factory).client('dax')
+        for r in resources:
+            try:
+                client.tag_resource(ResourceName=r[self.id_key], Tags=tags)
+            except ClientError as e:
+                if e.response['Error']['Code'] in (
+                        'ClusterNotFoundFault',
+                        'InvalidARNFault',
+                        'InvalidClusterStateFault',
+                        'InvalidParameterCombinationException',
```

Comment:
invalidparam* and tagnotfound can be removed from the except block, the later is only expected to be catched on delete tag.